Makefile

```diff
@@ -1,9 +1,15 @@
-.PHONY: help test dev-install install upgrade lint
+.PHONY: help clean test dev-install install upgrade lint
 
 help:				## display this help message
 	@echo "Please use \`make <target>' where <target> is one of"
 	@awk -F ':.*?## ' '/^[a-zA-Z]/ && NF==2 {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST) | sort
 
+clean:				## remove transient artifacts
+	rm -rf .*cache *.egg-info .coverage build/ htmlcov/
+	rm -rf .oep2-workspace/
+	rm -rf __pycache__ */__pycache__ */*/__pycache__
+	rm -rf *.pyc */*.pyc */*/*.pyc
```

Comment:
If possible, I'd like to use what's in the cookiecutter, which will go to all levels:

https://github.com/openedx/edx-cookiecutters/blob/f49b8beb757ae250747dea191e57e6ce037d9686/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/Makefile#L14

Reply:
(or just in general, take what's in the cookiecutter and then add on anything else we need to remove, like this `.oep2-workspace` thing)

Reply:
I knew there was a better way to do it....